src/player/features/volume/ui/VolumeBar.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { observer } from "mobx-react-lite";
+import React from "react";
+
+import { usePlayer } from "@/player/entities/player";
+import { Slider } from "@/shared/ui/inputs";
+import { VolumeFeature } from "../model/VolumeFeature";
+import { scaleVolumeToUnitRange } from "../utils/ScaleVolume";
+
+/**
+ * The VolumeBar component is a slider that allows the user to adjust the volume of the player.
+ */
+export const VolumeBar = observer(function VolumeBar() {
+  const player = usePlayer();
+  const [volumeFeature] = React.useState(() => new VolumeFeature({ player }));
+  const defaultVolume = 25;
+
+  return (
+    <Slider
+      defaultValue={[defaultVolume]}
+      onValueChange={([volume]) => {
+        volumeFeature.execute(scaleVolumeToUnitRange(volume));
+      }}
+    />
```

Comment:
Maybe I'm wrong but I think we need to bind `value` instead of `defaultValue`.

With `value` sets to `player.currentVolume` (we may need to convert our player `volume` values (comprised between `0` and `1`?) to the radix `Slider` value (comprised between `0` and `100`?)).

(This is where a `ViewModel` object might come in play: model values <-> view values).

---

In the meantime, do you think we can move the `scaleVolumeToUnitRange` transformation in the `VolumeFeature` `execute` method instead?